static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,485 @@
+              bars={10}
+              barHeight={20}
+              miseryLimit={1000}
+              totalUsers={500}
+              userMisery={300}
+              miserableUsers={200}
+            />
+          </UserMiseryPanelItem>
+          {field}
+        </div>
+      </Fragment>
+    );
+  }
+
+  renderUnknownPerformance() {
+    return (
+      <Fragment>
+        <div>
+          {/* Table description column */}
+          <PanelItem>{t('User Misery')}</PanelItem>
+        </div>
+        <div>
+          {/* TODO(kelly): placeholder data. will need to add user misery data in follow-up pr */}
+          <UserMiseryPanelItem>
+            <StyledUserMisery
+              bars={10}
+              barHeight={20}
+              miseryLimit={1000}
+              totalUsers={500}
+              userMisery={300}
+              miserableUsers={200}
+            />
+          </UserMiseryPanelItem>
+        </div>
+        <div>
+          {/* Table All Releases column */}
+          <UserMiseryPanelItem>
+            <StyledUserMisery
+              bars={10}
+              barHeight={20}
+              miseryLimit={1000}
+              totalUsers={500}
+              userMisery={300}
+              miserableUsers={200}
+            />
+          </UserMiseryPanelItem>
+        </div>
+        <div>
+          {/* Table This Release column */}
+          <UserMiseryPanelItem>
+            <StyledUserMisery
+              bars={10}
+              barHeight={20}
+              miseryLimit={1000}
+              totalUsers={500}
+              userMisery={300}
+              miserableUsers={200}
+            />
+          </UserMiseryPanelItem>
+        </div>
+      </Fragment>
+    );
+  }
+
+  render() {
+    const {project, organization} = this.props;
+    // Custom set the height so we don't have layout shift when results are loaded.
+    const loader = <LoadingIndicator style={{margin: '70px auto'}} />;
+
+    const title = FRONTEND_PLATFORMS.includes(project.platform as string)
+      ? t('Frontend Performance')
+      : BACKEND_PLATFORMS.includes(project.platform as string)
+      ? t('Backend Performance')
+      : MOBILE_PLATFORMS.includes(project.platform as string)
+      ? t('Mobile Performance')
+      : t('[Unknown] Performance');
+    const platformPerformance = FRONTEND_PLATFORMS.includes(project.platform as string)
+      ? this.renderFrontendPerformance()
+      : BACKEND_PLATFORMS.includes(project.platform as string)
+      ? this.renderBackendPerformance()
+      : MOBILE_PLATFORMS.includes(project.platform as string)
+      ? this.renderMobilePerformance()
+      : this.renderUnknownPerformance();
+
+    return (
+      <Fragment>
+        <HeadCellContainer>{title}</HeadCellContainer>
+        {title.includes('Unknown') ? (
+          <StyledAlert type="warning" icon={<IconWarning size="md" />} system>
+            For more performance metrics, specify which platform this project is using in{' '}
```

Comment:
Should this be translated?

Reply:
I think this change still hasn't been pushed?

Reply:
updated with the boolean check from the other comment :)